content/rs/installing-upgrading/_index.md

```diff
@@ -70,42 +57,40 @@ Before installing, review these notes:
 
-```sh
-sudo /opt/redislabs/sbin/prepare_flash.sh
-```
+    Use the `prepare_flash` script to prepare and format flash memory:
 
-This command finds all of the unformatted disks and mounts them as RAID partitions in `/var/opt/redislabs/flash`.
+    ```sh
+    sudo /opt/redislabs/sbin/prepare_flash.sh
+    ```
 
-To verify the disk configuration, run:
+    This script finds unformatted disks and mounts them as RAID partitions in `/var/opt/redislabs/flash`.
 
-```sh
-sudo lsblk
-```
-    {{% /expand %}}
+    To verify the disk configuration, run:
+
+    ```sh
+    sudo lsblk
+    ```
 
 - [Disable Linux swap]({{< relref "/rs/installing-upgrading/configuring/linux-swap.md" >}}) on all cluster nodes.
-- Make sure that you have root-level access to each node, either directly or with sudo.
+- Make sure that you have root-level access to each node, either directly or with `sudo`
 - If you require the `redislabs` UID (user ID) and GID (group ID) numbers to be the same on all the nodes, create the `redislabs` user and group with the required numbers on each node.
 - When port 53 is in use, the installation fails. This is known to happen in
-    default Ubuntu 18.04 installations in which systemd-resolved (DNS server) is running.
+    default Ubuntu 18.04 installations in which `systemd-resolved` (DNS server) is running.
+
     To work around this issue, change the system configuration to make this port available
-    before installation.
+    before installation.  Here's one way to do so:
 
-    {{% expand "Example steps to resolve the port 53 conflict:" %}}
-1. Run: `sudo vi /etc/systemd/resolved.conf`
-1. Add `DNSStubListener=no` as the last line in the file and save the file.
-1. Run: `sudo mv /etc/resolv.conf /etc/resolv.conf.orig`
-1. Run: `sudo ln -s /run/systemd/resolve/resolv.conf /etc/resolv.conf`
-1. Run: `sudo service systemd-resolved restart`
-    {{% /expand %}}
+    1. Run: `sudo vi /etc/systemd/resolved.conf`
+    1. Add `DNSStubListener=no` as the last line in the file and save the file.
+    1. Run: `sudo mv /etc/resolv.conf /etc/resolv.conf.orig`
+    1. Run: `sudo ln -s /run/systemd/resolve/resolv.conf /etc/resolv.conf`
+    1. Run: `sudo service systemd-resolved restart`
 
-- Make sure that the OS is not using ports in the [range that Redis assigns to databases]({{< relref "/rs/administering/designing-production/networking/port-configurations.md" >}}).
+- Make sure that the operating system isn't using ports [range Redis assigns to databases]({{< relref "/rs/administering/designing-production/networking/port-configurations.md" >}}).
 
-    We recommend that you restrict the OS from using Redis ports range in `/etc/sysctl.conf` with `net.ipv4.ip_local_port_range = 30000 65535'.
+    We recommend updating `/etc/sysctl.conf` to include `net.ipv4.ip_local_port_range = 30000 65535'.
```

Comment:
Will it be better to explain why? To avoid port collision?

Reply:
Fixed.